Update index.md Bas document changes (#606)
HL7/fhircast-docs: input/pagecontent/2-5-EventNotification.md

### Event Notification

The event notification message includes the following fields:

{:.grid}
Field       | Optionality | Type | Description
----------- | ----------- | ---- | ------------

> `timestamp` | Required    | *string* | ISO 8601-2 timestamp in UTC describing the time from the Request Event Notification.

Reviewer: ```suggestion
`timestamp` | Required    | *string* | ISO 8601-2 timestamp in UTC describing the time at which the event occurred.

```
I don't understand the proposed change and it feels like it's a new, different definition of this field, which we oughtn't do. Reverting.

### Event Notification

The event notification message includes the following fields:

{:.grid}
Field       | Optionality | Type | Description
----------- | ----------- | ---- | ------------
`timestamp` | Required    | *string* | ISO 8601-2 timestamp in UTC describing the time at which the event occurred.

> 

Reviewer: ```suggestion

```
remove extraneous whitespace
